.github/workflows/ds-storybook-deploy.yml

```diff
@@ -0,0 +1,41 @@
+name: Deploy Design System Storybook
+
+on:
+  push:
+    paths:
+      - "packages/design-system/**"
+    branches:
+      - "main"
+
+# Change the working directory for all the jobs in this workflow
+defaults:
+  run:
+    working-directory: packages/design-system
+
+jobs:
+  deploy-storybook:
+    runs-on: ubuntu-latest
+    steps:
+      # Checkout the code
+      - uses: actions/checkout@v3
+
+      # Setup node with version 16.x and NPM registry url
+      - uses: actions/setup-node@v1
+        with:
+          node-version: "16.x"
+          registry-url: "https://registry.npmjs.org"
+
+      # yarn build to build the project
+      - name: Build package
+        run: yarn build-storybook
+
+      - name: configure aws credentials
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          role-to-assume: arn:aws:iam::391311798816:role/design-systems-github-actions-deploy-s3
```

Comment:
arn:aws:iam::391311798816? can it be a security concern?

Reply:
Account IDs aren't sensitive information, and should be okay here.

@mohanarpit, if your opinion on this is different, we can move it to a secret, by I don't think account ID should be a secret.